custom_platforms.js

```diff
@@ -85,6 +85,9 @@ module.exports = {
         "path": "ios",
         "configuration": {
             "removeRulePatterns": [
+                "\\$extension",
+                ",extension",
+                "\\$remove(param|header)",
```

Comment:
`removeparam` and `removeheader` can be also used with other modifiers, for example:
```adblock
||example.org^$xmlhttprequest,removeparam=param
||example.org^$xmlhttprequest,removeheader=location
```
so I guess that these kind of rules should be also excluded.

Reply:
In fact, I only see the rules like below, but maybe it will be useful for future ones.
```adblock
||example.org^$removeheader=location
```
@AdamWr what about this? Seems it removes the rules you mentioned fine.
```
(\\$|,)remove(param|header)
```

Reply:
As I can see on those files, there is no difference, but the pattern above seems a bit specific, so I've replaced it.